### PR TITLE
Remove byteorder-dependency

### PR DESCRIPTION
**Pull request by lukaslueg**
This removes the `byteorder`-dependency from the `web`-component, replacing it with the `stdlib`-implementations that were stabilized in 1.32 and 1.34. Dropping a dependency is worth the change IMHO.

Notice that the calls to `.unwrap()` do not change previous behavior regarding panics.

### CODE REVIEW

**Review by leizzer — COMMENTED** [review]
LGTM
I don't see any issue with it. Would like to hear more from others before approving.

**Comment by dewey4iv** [comment]
I took a peek; but would like the opportunity to take a closer look this evening.

File: actix-http/src/ws/frame.rs [hunk 1]
```diff
@@ -201,10 +204,7 @@ impl Parser {
         let payload = match reason {
             None => Vec::new(),
             Some(reason) => {
-                let mut code_bytes = [0; 2];
-                NetworkEndian::write_u16(&mut code_bytes, reason.code.into());
-
-                let mut payload = Vec::from(&code_bytes[..]);
+                let mut payload = Into::<u16>::into(reason.code).to_be_bytes().to_vec();
```
Comment (fafhrd91) [inline actix-http/src/ws/frame.rs:207]: this is heap allocation. is it possible to avoid vec creation?

i missed old line, no worries about this.

**Review by fafhrd91 — APPROVED** [review]

**Comment by dewey4iv** [comment]
LGTM

Feels good removing dependencies.

**Comment by dewey4iv** [comment]
I feel good merging this. @fafhrd91 - any objections?

**Review by fafhrd91 — APPROVED** [review]

**Comment by fafhrd91** [comment]
No objections, just need changes.md entry

**Comment by leizzer** [comment]
@lukaslueg can you update https://github.com/actix/actix-web/blob/master/CHANGES.md under 1.0.6?

**Review by leizzer — CHANGES_REQUESTED** [review]
Update Changes.md

**Comment by lukaslueg** [comment]
Updated. I misspoke earlier, dropping `byteorder` only affects `actix-http`, so I changed it's own `CHANGES.md`

**Review by leizzer — APPROVED** [review]

**Review by fafhrd91 — APPROVED** [review]

**Pull request merged**